app/src/main/java/edu/umd/cmsc436/mstestsuite/MainActivity.java

```diff
@@ -246,8 +264,8 @@ public void onClick(View v) {
 
     @Override
     public void onBackPressed() {
-        if (mPresenter.onBackPressed()) {
+        //if (mPresenter.onBackPressed()) {
```

Comment:
This code is important and should be left in place, it enables the user to go back from practice mode